TAF_EasyRest/TestFramework/Pages/SignUpPage.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
We don't use lines 1-5. Please remove them in the next pull request.

Reply:
fixed